public/javascripts/Gallery/css/expanded-view.css

```diff
@@ -58,6 +58,13 @@
     font-family: 'raleway-bold', sans-serif;
     font-size: 1.5vw;
     color: #2d2a3f;
+    display: flex;
+    align-items: center;
+    column-gap: 10px;
+}
+
+.gallery-expanded-view-header-label {
```

Comment:
This CSS class isn't used anywhere, is it? I think that since you removed the icon from the header, all the edits from this file can likely be removed?